### DISCUSSION TITLE
argcd app actions list - not listing controller created/managed "Deployment" kinds

### DISCUSSION THREAD

**Discussion by pon-raj** [argoproj/argo-cd#13014] [Q&A]
Hello,
"argcd app actions list" command does not list "Deployment" kinds that are created/managed by controllers. It returns empty list.
--loglevel debug is not adding any additional information to troubleshoot.

The same command works if the resources are not managed by controllers.

Please help me to troubleshoot the issue.

Thanks.

**Comment by e30532** [comment]
I also observed the same issue locally.

If we create an application(deployment) through CRD(Custom Resrouce Definition), which is Open Liberty Operator in my case, on OpenShift Container Platform v4.12, "actions list" doesn't return anything. Thus, we can't do the restart operation against the deployment via CLI. In the Web console, I was able to restart the deployment in the Argo CD web console.

Here is what I did in my local test system.
https://github.com/e30532/argocd/blob/main/README.md

And as you see below, actions list returns nothing.

% argocd app actions list myola
GROUP  KIND  NAME  ACTION  DISABLED
% 

Hopefully there is a way to list/restart the application/deployment in CLI even if it's deployed as CRD.
